gerritlab/merge_request.py

```diff
@@ -132,6 +134,15 @@ def delete(self, delete_source_branch=False):
         if delete_source_branch:
             self._remote.push(refspec=(":{}".format(self._source_branch)))
 
+    def reload(self):
```

Comment:
💡 future idea: I wonder if we'll want a command line argument someday to do this; e.g., `--reload-all`

Reply:
Yes, will add a command line argument for this in next PR.